internal/db/migrations-tern/006_create_table_blueprints.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS blueprints
+(
+    id uuid PRIMARY KEY,
+    org_id varchar NOT NULL CHECK (coalesce(trim(org_id), '') != ''),
+    account_number varchar NOT NULL CHECK (coalesce(trim(account_number), '') != ''),
+    name VARCHAR(100) NOT NULL,
+    description VARCHAR(250),
+    body_version INTEGER NOT NULL,
+    body JSONB NOT NULL,
```

Comment:
This is not blueprint version from the users perspective, this is body (payload) version which will be increased as soon as incompatible JSON change will be done. This must be stored outside of the JSON itself because only then you can effectively pick the proper type, `BlueprintV1` in this case, and unmarshal data into specific type.

Blueprint versioning will be done in a separate PR and that field will be likely called simply `version`, or we might end up only having `parent uuid` column. We talked about this with @avitova today on techdiss, it needs more research on what will be more appropriate for this case.

Reply:
Ah! Gotcha, than ignore my comment, this makes sense!

Reply:
JSON unmarshaling is very forgiving though, so you could just grab the version and then unmarshal it again into the appropriate struct.

Once we build versioning on top we need a way to group the blueprints indeed.